tests/test_rocket.py

```diff
@@ -509,3 +532,25 @@ def test_set_rail_button(calisto):
     assert calisto.rail_buttons[0].component.buttons_distance + calisto.rail_buttons[
         0
     ].position == pytest.approx(0.2, 1e-12)
+
+
+def test_evaluate_total_mass(calisto_motorless):
+    """Tests the evaluate_total_mass method of the Rocket class.
+    Both with respect to return instances and expected behaviour.
+    Parameters
+    ----------
+    calisto_motorless : Rocket instance
+        A predefined instance of a Rocket without a motor, used as a base for testing.
+    """
+    assert isinstance(calisto_motorless.evaluate_total_mass(), Function)
```

Comment:
Good idea to assert this part of the code. But I'd suggest to also check for expected values.